Bento/Views/BentoReusableView.swift

```diff
@@ -6,11 +6,15 @@ protocol BentoReusableView: AnyObject {
 
 extension BentoReusableView {
     func bind(_ component: AnyRenderable?) {
+        let oldComponent = self.component
         self.component = component
+
         if let component = component {
             let renderingView: UIView
 
-            if let view = containedView, type(of: view) == component.viewType {
+            if let oldComponent = oldComponent,
+               let view = containedView,
+               oldComponent.reusabilityHintCombiner.isCompatible(with: component.reusabilityHintCombiner) {
```

Comment:
This should not usually happen, should it?

Reply:
All visible items hit this path when the state changes.

Reply:
I mean `oldComponent.reusabilityHintCombiner.isCompatible(with: component.reusabilityHintCombiner)` to be `false`. As it'f false it means that we are recreating the view, right?

Reply:
yeh